Error: must provide pattern

when I run `npm run commit` this is what my log looks like:

```
$ npm run commit

> backoff@0.0.0 precommit /Users/aaronreisman/Workspace/lifeiscontent/backoff
> lint-staged

 ❯ Running tasks for {src,test}/**/*.ts
   ✖ prettier --write --no-semi  --single-quote
     → Error: must provide pattern
     git add
✖ prettier --write --no-semi  --single-quote found some errors. Please fix them and try committing again.

Unable to expand glob patterns:  /Users/aaronreisman/Workspace/lifeiscontent/backoff/test/api.test.ts /Users/aaronreisman/Workspace/lifeiscontent/backoff/src/strategy/index.ts /Users/aaronreisman/Workspace/lifeiscontent/backoff/src/strategy/fibonacci.ts /Users/aaronreisman/Workspace/lifeiscontent/backoff/src/strategy/exponential.ts /Users/aaronreisman/Workspace/lifeiscontent/backoff/src/index.ts /Users/aaronreisman/Workspace/lifeiscontent/backoff/src/function-call.ts /Users/aaronreisman/Workspace/lifeiscontent/backoff/src/backoff.ts !**/node_modules/** !./node_modules/**
Error: must provide pattern

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! backoff@0.0.0 precommit: `lint-staged`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the backoff@0.0.0 precommit script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/aaronreisman/.npm/_logs/2017-10-09T17_28_07_024Z-debug.log
aaronreisman at Aarons-MacBook-Pro in ~/Workspace/lifeiscontent/backoff on master▲
```

--- a/package.json
+++ b/package.json
@@ -36,7 +36,7 @@
   },
   "lint-staged": {
     "{src,test}/**/*.ts": [
-      "prettier --write --no-semi  --single-quote",
+      "prettier --write --no-semi --single-quote",
       "git add"
     ]
   },

